designs/black-parrot/descriptor.yaml

```
@@ -0,0 +1,432 @@
+    - src/bp/bp_be_dcache_wbuf.sv
+    - src/bp/bp_fe_ras.sv
+    - src/bp/bp_fe_bht.sv
+    - src/bp/bp_fe_btb.sv
+    - src/bp/bp_fe_controller.sv
+    - src/bp/bp_fe_icache.sv
+    - src/bp/bp_fe_scan.sv
+    - src/bp/bp_fe_pc_gen.sv
+    - src/bp/bp_fe_realigner.sv
+    - src/bp/bp_fe_top.sv
+    - src/bp/bp_lce.sv
+    - src/bp/bp_lce_req.sv
+    - src/bp/bp_lce_cmd.sv
+    - src/bp/bp_me_bedrock_register.sv
+    - src/bp/bp_me_cache_controller.sv
+    - src/bp/bp_me_dram_hash_decode.sv
+    - src/bp/bp_me_dram_hash_encode.sv
+    - src/bp/bp_me_cache_slice.sv
+    - src/bp/bp_me_cfg_slice.sv
+    - src/bp/bp_me_clint_slice.sv
+    - src/bp/bp_me_loopback.sv
+    - src/bp/bp_cce.sv
+    - src/bp/bp_cce_alu.sv
+    - src/bp/bp_cce_arbitrate.sv
+    - src/bp/bp_cce_branch.sv
+    - src/bp/bp_cce_dir.sv
+    - src/bp/bp_cce_dir_lru_extract.sv
+    - src/bp/bp_cce_dir_segment.sv
+    - src/bp/bp_cce_dir_tag_checker.sv
+    - src/bp/bp_cce_gad.sv
+    - src/bp/bp_cce_inst_decode.sv
+    - src/bp/bp_cce_inst_predecode.sv
+    - src/bp/bp_cce_inst_ram.sv
+    - src/bp/bp_cce_inst_stall.sv
+    - src/bp/bp_cce_msg.sv
+    - src/bp/bp_cce_pending_bits.sv
+    - src/bp/bp_cce_pma.sv
+    - src/bp/bp_cce_reg.sv
+    - src/bp/bp_cce_spec_bits.sv
+    - src/bp/bp_cce_src_sel.sv
+    - src/bp/bp_io_cce.sv
+    - src/bp/bp_cce_fsm.sv
+    - src/bp/bp_cce_wrapper.sv
+    - src/bp/bp_bedrock_size_to_len.sv
+    - src/bp/bp_uce.sv
+    - src/bp/bp_me_addr_to_cce_id.sv
+    - src/bp/bp_me_cce_id_to_cord.sv
+    - src/bp/bp_me_cord_to_id.sv
+    - src/bp/bp_me_lce_id_to_cord.sv
+    - src/bp/bp_me_stream_pump.sv
+    - src/bp/bp_me_stream_pump_in.sv
+    - src/bp/bp_me_stream_pump_out.sv
+    - src/bp/bp_me_stream_pump_control.sv
+    - src/bp/bp_me_stream_to_wormhole.sv
+    - src/bp/bp_me_wormhole_header_encode.sv
+    - src/bp/bp_me_wormhole_to_stream.sv
+    - src/bp/bp_me_wormhole_stream_control.sv
+    - src/bp/bp_me_xbar_stream.sv
+    - src/bp/bp_me_stream_gearbox.sv
+    - src/bp/bp_nd_socket.sv
+    - src/bp/bp_cacc_vdp.sv
+    - src/bp/bp_cacc_tile.sv
+    - src/bp/bp_cacc_tile_node.sv
+    - src/bp/bp_cacc_complex.sv
+    - src/bp/bp_sacc_vdp.sv
+    - src/bp/bp_sacc_scratchpad.sv
+    - src/bp/bp_sacc_tile.sv
+    - src/bp/bp_sacc_tile_node.sv
+    - src/bp/bp_sacc_complex.sv
+    - src/bp/bp_core.sv
+    - src/bp/bp_core_lite.sv
+    - src/bp/bp_core_minimal.sv
+    - src/bp/bp_core_complex.sv
+    - src/bp/bp_l2e_tile.sv
+    - src/bp/bp_l2e_tile_node.sv
+    - src/bp/bp_io_complex.sv
+    - src/bp/bp_io_link_to_lce.sv
+    - src/bp/bp_io_tile.sv
+    - src/bp/bp_io_tile_node.sv
+    - src/bp/bp_mem_complex.sv
+    - src/bp/bp_multicore.sv
+    - src/bp/bp_unicore.sv
+    - src/bp/bp_unicore_lite.sv
+    - src/bp/bp_core_tile.sv
+    - src/bp/bp_core_tile_node.sv
+    - src/bp/bp_processor.sv
+    - src/bp/bsg_async_noc_link.sv
+    - src/bp/bsg_dff_sync_read.sv
+    - src/bp/bsg_rom_param.sv
+    - src/bp/wrapper.sv
+    #Basejump Memory Files
```

```suggestion
    # Basejump Memory Files
```
Though I'm not sure all the following are "Memory Files"? It's ok to strip these comments if the separation is not entirely clean.